src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -46,6 +46,12 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
     for (size_t hindex = 0; hindex < UARTDEV_COUNT_MAX; hindex++) {
```

Comment:
BTW: `hardware->reg` is set for each `uartHardware` entry, co it may overflow `uartDevice` when user changes pin config ...